draganddrop/draganddrop/MainPage.xaml.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
```

Comment:
Oh yeah, I understand.  We are going to skip overwriting those samples who have customized any of the shared files or are not using the template.

From: Raymond Chen [mailto:notifications@github.com]
Sent: Monday, June 15, 2015 9:23 AM
To: Microsoft/Windows-universal-samples
Cc: Mike Henderlight
Subject: Re: [Windows-universal-samples] drag and drop sample (#25)


In draganddrop/draganddrop/MainPage.xaml.cs<https://github.com/Microsoft/Windows-universal-samples/pull/25#discussion_r32438382>:

> @@ -0,0 +1,62 @@

> +﻿using System;

This happened, for example, to the FeedReader sample. It had a customized default.js file that got overwritten when we deployed new JS templates. See f59dbfe<https://github.com/Microsoft/Windows-universal-samples/commit/f59dbfe63c5e0c3f73d5db8d70abd2fabd62ed08> which introduced the problem, 74d496d<https://github.com/Microsoft/Windows-universal-samples/commit/74d496d95c9f9a43a3bf3ca581bb8b8f0b78151c> which fixed it, and c697adc<https://github.com/Microsoft/Windows-universal-samples/commit/c697adc1a7f3d2cc7aa5fabe83f0a73e35078fd7> which renamed default.js to main.js so it won't get inadvertently stomped by future template changes.

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/Windows-universal-samples/pull/25/files#r32438382>.